specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure IoT Central",
+    "description": "Azure IoT Central is a service that makes it easy to connect, monitor, and manage your IoT devices at scale.",
+    "version": "1.0.0"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{subdomain}.{baseDomain}",
+    "parameters": [
+      {
+        "$ref": "#/parameters/BaseDomain"
+      },
+      {
+        "$ref": "#/parameters/Subdomain"
+      }
+    ]
+  },
+  "basePath": "/api/v1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "definitions": {
+    "ApiToken": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Permission"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "Unique ID of the API token.",
+              "type": "string",
+              "readOnly": true
+            },
+            "token": {
+              "description": "Value of the API token.",
+              "type": "string",
+              "readOnly": true
+            },
+            "expiry": {
+              "description": "String-formatted date representing the time when the token expires.",
+              "type": "string",
+              "format": "date-time"
+            }
+          }
+        }
+      ]
+    },
+    "ApiTokenCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of API tokens.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiToken"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of API tokens.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "Attestation": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Type of the attestation.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "type"
+      ],
+      "discriminator": "type"
+    },
+    "Collection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection of objects.",
```

Comment:
Could this be more descriptive? `objects` is very ambiguous.

Reply:
We shouldn't have collection for attestation, there might some mistake on my swagger, will remove this collection defnition.

Reply:
Updated comments.